assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -32,6 +32,8 @@ che.workspace.storage=${che.home}/workspaces
 # Your projects are synchronized from the Che server into the machine running each
 # workspace. This is the directory in the machine where your projects are placed.
 che.workspace.projects.storage=/projects
+# Used when devfile k8s/os type components requests project PVC creation
```

Comment:
Maybe it may sense to mention that projects PVC will have such size only in case of `unique` and `perWorkspace` PVC strategy. In case of common PVC strategy, it will be rewritten with https://github.com/eclipse/che/blob/f35e870a88c4fdb60c85095d747e0e2360039ccf/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L447